doc/nrf/dm_ncs_distro.rst

```diff
@@ -0,0 +1,64 @@
+.. _dm_ncs_distro:
+
+Redistributing the |NCS|
+########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This section is intended for users that would like to re-distribute the |NCS| with additions or changes to it.
+It can be useful if you are a software or hardware vendor that would like to augment the |NCS| with additional functionality, and still benefit from the infrastructure it provides.
+
+The contents of this section is not comprehensive.
+You can rather consider it as suggestions for extending the |NCS|.
+You can use other mechanisms as long as they comply with the text in the :file:`LICENSE` file located in the root of every Git repository in the |NCS|.
+If you need any help, reach out to Nordic Semiconductor.
+
+Getting added to the main manifest
+**********************************
+
+The simplest way to become part of the |NCS| is to be included in its official `west manifest file`_.
+This way, your own :ref:`module repository <zephyr:modules>` will automatically be part of the |NCS| official distribution and will be included in all releases.
+
+This solution is reserved for selected vendors and requires a commercial agreement with Nordic Semiconductor.
```

Comment:
I am new to this NCS and following might be covered: Does the commercial agreement define testing and compatibility responsibilities?  Is Nordic some how responsible on verifying compatibility or does the redistributor take the responsibility?

If this page is for the contributors, about "how to become a redistributor" or "How to be part of nRF Connect SDK delivery". Do we have some page for the users describing that SDK contains parts of other contributors... what are nordic own and what parts are 3rd party?

Reply:
>I am new to this NCS and following might be covered: Does the commercial agreement define testing and compatibility responsibilities? Is Nordic some how responsible on verifying compatibility or does the redistributor take the responsibility?

Very good question. There is no generic answer to this, it really depends on each individual agreement with Nordic how testing, compatibility and maintenance are handled.

>If this page is for the contributors, about "how to become a redistributor" or "How to be part of nRF Connect SDK delivery

This page is for those who want to redistribute nRF Connect SDK, not just use it to develop a product.

>Do we have some page for the users describing that SDK contains parts of other contributors... what are nordic own and what parts are 3rd party?

We don't specifically, but it's relatively simple to find out based on the [manifest](https://github.com/nrfconnect/sdk-nrf/blob/main/west.yml) or the [repo revision table](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/introduction.html#ncs-repository-revisions). For each repository you can see whether it's Nordic-developed or 3rd-party code by looking at the license and description.